includes/Infrastructure/WooCommerce/Gateway/Frontend/PayLaterGateway.php

```diff
@@ -44,8 +46,15 @@ public function validate_fields(): bool {
 		);
 
 		// phpcs:ignore
-		if ( $_POST['alma_deferred'] && ! $this->check_values( $_POST['alma_deferred'],
-			array( '15_0', '30_0', '45_0', '0_1', '0_2', '0_3' )
+		if ( $_POST['alma_plan_key'] && ! $this->check_values( $_POST['alma_plan_key'],
```

Comment:
I think is better to use a function isPayLater to avoid an array of each plan, like Credit ?